### PR TITLE
support multiple parallel builds.

### PR DESCRIPTION
**Pull request by dangula**

### CODE REVIEW

**Comment by dangula** [comment]
E2E tests are running in true docker in docker now (vs docker on docker), this should enable us to run multiple builds simultaneously .

File: e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh [hunk 1]
```diff
@@ -140,15 +140,6 @@ function dind::prepare-sys-mounts {
     if [[ -d /lib/modules ]]; then
       sys_volume_args+=(-v /lib/modules:/lib/modules)
     fi
-    if [[ -d /dev ]]; then
```
Comment (travisn) [inline e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh]: did we pick up these changes from external or did we make changes to this script?

File: e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh [hunk 2]
```diff
@@ -399,6 +390,9 @@ function dind::run {
          --hostname "${container_name}" \
          -l mirantis.kubeadm_dind_cluster \
          -v ${volume_name}:/dind \
+         -v /lib/modules:/lib/modules \
+         -v /sbin/modprobe:/sbin/modprobe \
```
Comment (travisn) [inline e2e/framework/manager/scripts/rook-dind-cluster-v1.6.sh]: was this the core cause of block tests failing?

File: e2e/scripts/smoke_test.sh [hunk 3]
```diff
@@ -61,6 +56,34 @@ rook_infra::run_test() {
     rc=$?; if [[ $rc != 0 ]]; then set -e; fi
 }
 
+rook_infra::try_copy_docker_image_to_rook_infra() {
```
Comment (travisn) [inline e2e/scripts/smoke_test.sh]: why do we need to load images explicitly? Aren't they already loaded from the build phase?

File: e2e/scripts/smoke_test.sh [hunk 4]
```diff
@@ -108,15 +132,15 @@ rook_infra::cleanup() {
 { #try
 
     rook_infra::create
-    sleep 5
+    sleep 10    #need to add code to retry docker info till success
```
Comment (travisn) [inline e2e/scripts/smoke_test.sh]: can you add this check? seems like we need to fix these sleeps instead of letting them perpetuate

Comment (dangula) [inline e2e/scripts/smoke_test.sh]: I didn't realize this check was here. I have removed it and we are making sure the container is in running state before we proceed.

**Pull request merged**